scripts/backpressure_report/README.md

```diff
@@ -0,0 +1,45 @@
+# Backpressure Report
+
+This `backpressure_report` Python project allows for measuring the amount of time Cromwell runner instances spend in a
+high I/O state which triggers internal backpressure. While in this high I/O state the Cromwell runners will not hand out
+job execution or restart check tokens, so job starts and restarts will be slowed until I/O returns to normal levels.
+
+## Running
+
+Usage:
+
+```shell
+python -m backpressure_report.main <files with JSON formatted Logs Explorer logs>
```

Comment:
I'm having trouble running this, first issue I hit is that dateutil is not installed. Typically in Python you would create a `requirements.txt` file in this top directory containing the packages needed by the program, that users can install via Pip. More here: https://pip.pypa.io/en/stable/user_guide/

Reply:
I added a `setup.py` to handle this dependency and also constrain the Python version for f string support, along with some installation instructions in the README.md.